shell/platform/linux/fl_renderer.cc

```diff
@@ -25,12 +25,24 @@ static void fl_renderer_class_init(FlRendererClass* klass) {}
 
 static void fl_renderer_init(FlRenderer* self) {}
 
-gboolean fl_renderer_setup(FlRenderer* self, GError** error) {
```

Comment:
All these functions that were converted to static need comments above them describing what they do.